tests/Integration/Resources/Computing/Test_ApptainerCE.py

```diff
@@ -0,0 +1,90 @@
+#!/bin/env python
+
+""" This integration test is for "Inner" Computing Element ApptainerComputingElement
+    This test is here and not in the unit tests because it requires apptainer to be installed.
+"""
+
+import six
+import os
+import shutil
+
+from DIRAC import gLogger
+from DIRAC.tests.Utilities.utils import find_all
+
+from DIRAC.Resources.Computing.test.Test_PoolComputingElement import jobScript, _stopJob
+from DIRAC.WorkloadManagementSystem.Utilities.Utils import createJobWrapper
+
+# sut
+from DIRAC.Resources.Computing.ApptainerComputingElement import ApptainerComputingElement
+
+
+gLogger.setLevel("DEBUG")
+fj = find_all("pilot.json", "../", "tests/Integration/Resources/Computing")[0]
+fc = find_all("pilot.cfg.test", "../", "tests/Integration/Resources/Computing")[0]
+
+
+def test_submitJob():
+    shutil.copy(fj, os.curdir)
+    shutil.copyfile(fc, os.path.join(os.curdir, "pilot.cfg"))
+    with open("testJob.py", "w") as execFile:
+        execFile.write(jobScript % "1")
+    os.chmod("testJob.py", 0o755)
+
+    ce = ApptainerComputingElement("ApptainerComputingElement")
+    res = ce.submitJob("testJob.py", None)
+    assert res["OK"] is False
+    assert res["ReschedulePayload"] is True
+    res = ce.getCEStatus()
+    assert res["OK"] is True
+    if six.PY2:
+        assert res["SubmittedJobs"] == 0
+    else:
+        assert res["SubmittedJobs"] == 1
```

Comment:
Why?

Reply:
This is a copy and paste of the SingularityCE test so I have no idea, I defer to the original author of the change 😅 https://github.com/DIRACGrid/DIRAC/commit/96de9277f8520f62d9dfbf14ba46dc5e38d4f12d#diff-3ed8e61b9808b5f01275986e1b4401914daf0ce427d6ebd4f809f695948fe43a

Reply:
AH, it must be correct then! 😄 

Maybe because it's only py3? https://github.com/DIRACGrid/DIRAC/pull/5716